lib/src/main/java/module-info.java

```diff
@@ -0,0 +1,9 @@
+module com.auth0.java_jwt {
```

Comment:
Why did you rename the module from `com.auth0.jwt` to `com.auth0.java_jwt` though?
At least the JLS suggests to use the name of the principal exported package, which is `com.auth0.jwt`.